### DISCUSSION TITLE
[BUG] Fix supported syntax

### DISCUSSION THREAD

**Discussion by dani-petrov** [JSQLParser/JSqlParser#2111] [Q&A]
Hello, I will attach a screenshot of the currently supported syntax for Alter Table -> modify column:
<img width="1067" alt="Screenshot 2024-11-19 at 9 59 46" src="https://github.com/user-attachments/assets/9e8ffde4-8299-47a8-b654-14a7c832af4c">
Looking at the circled part, the correct syntax for a MODIFY [COLUMN] statement is together with [IF EXISTS] and not [IF NOT EXISTS]. This leads to failed parsing. Is it possible to allow both IF NOT EXISTS and IF EXISTS?
Thank you!

**Comment by manticore-projects** [comment]
Greetings! Your observation is certainly correct and I will have to take a look at that.
